### DISCUSSION TITLE
some hypothetical mining questions - hardware specific

### DISCUSSION THREAD

**Discussion by tosufever** [veryengineer/collab#16] [Ideas]
https://github.com/pooler/cpuminer

a decent thought exercise relevant to this project:
https://www.tomshardware.com/how-to/mine-cryptocurrency-raspberry-pi 

to be continued…

is there a way to draft discussions? otherwise this is just a placeholder that forces me to circle back.

https://raspberrypi.stackexchange.com/questions/116115/raspberry-pi-4-gpu-math
https://www.iqt.org/eating-every-last-slice-of-your-raspberry-pi-gpu-offload-on-a-pi4/

**Comment by patricklodder** [comment]
> is there a way to draft discussions?

Yeah you just did it. Now I ask: did you mean CPU mining? 

And then you can update the top post as we go 😁

**Reply by tosufever** [reply]
two quick questions. 
first, i need a recommendation for a miner. preferably something available used / ebay or newegg. just something to walkthrough the whole process at home for fun; it doesn't have to be profitable. newegg has some L3++ referbs on a boat from china for $330. is that a good fit to tinker with? 


second, what is the general consensus on single asic via usb?  
https://shop.futurebit.io/collections/all 
![usb asic](https://user-images.githubusercontent.com/108495285/221208417-65ff4bc2-6427-4559-aaeb-f1559420e14a.PNG)

**Reply by patricklodder** [reply]
L3s are common so that would be good for tinkering, USB miners too...
